packages/canvas-tokens-docs/.storybook/Label.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+const flexStyles = {
+  display: 'flex',
+  alignItems: 'center',
```

Comment:
maybe we should use `start` to align vertically, so label will stay on the same place when label text is split

Reply:
i like